### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by ikwuoz**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by ikwuoz** [comment]
pr review test comment

**Review by ikwuoz — COMMENTED** [review]
test review comment

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (ikwuoz) [inline index.html]: test suggestion.

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (nuel-ikwuoma) [inline index.html:50]: test suggestion

Comment (nuel-ikwuoma) [reply index.html:50]: test resolved

File: index.html (hunk 1 above)
Comment (ikwuoz) [inline index.html]: test commit suggestion

Comment (nuel-ikwuoma) [reply index.html:50]: test commit suggestion [review]

**Review by ikwuoz — COMMENTED** [review]
test finish review.

File: index.html (hunk 1 above)
Comment (ikwuoz) [inline index.html]:
```suggestion
<h3 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h3>
```

**Pull request merged**